packages/surveycto/src/Adaptor.js

```diff
@@ -24,92 +21,133 @@ export function execute(...operations) {
   };
 
   return state => {
-    state.configuration.baseUrl = 'https://'.concat(
-      state.configuration.instanceName,
-      '.surveycto.com/api/v1'
-    );
     return commonExecute(...operations)({ ...initialState, ...state });
   };
 }
 
 /**
- * Make a GET request and POST it somewhere else
- * @example
- * execute(
- *   fetch(params)
- * )(state)
+ * Options provided to the HTTP request
+ * @typedef {Object} FormSubmissionOptions
+ * @property {string} [date=0] - Form completion or submission date. Default to `0` which will return all submission data
+ * @property {string} [format='json'] - Form response type, It can be in `csv` or `json`. Default to `json`
+ * @property {string} status - (Opt)Review status. Can be either, `approved`, `rejected`, `pending` or combine eg `approved|rejected`.
+ */
+
+/**
+ * Fetch form submissions
+ * @example <caption>Using state lazy load</caption>
+ * fetchSubmissions($.formId || 'test', { date: '2024-04-18' });
```

Comment:
I am not sure I'm ready to add lazy state into our official docs yet. Tricky one.

I think for your should do `fetchSubmissions(state.formId || 'test', { date: '2024-04-18' });`. It's close enough.